node/narwhal/src/helpers/committee.rs

```diff
@@ -444,4 +500,28 @@ pub mod prop_tests {
         // (2f + 1) + (f + 1) - 1 = 3f + 1 = N
         assert_eq!(quorum_threshold + availability_threshold - 1, total_stake);
     }
+
+    #[proptest]
+    fn invalid_stakes(#[strategy(too_low_stake_committee())] committee: Result<Committee<CurrentNetwork>>) {
+        assert!(committee.is_err());
+        if let Err(err) = committee {
+            assert_eq!(err.to_string().as_str(), "All members must have sufficient stake");
+        }
+    }
+
+    #[proptest]
+    fn invalid_member_count(#[strategy(too_small_committee())] committee: Result<Committee<CurrentNetwork>>) {
+        assert!(committee.is_err());
+        if let Err(err) = committee {
+            assert_eq!(err.to_string().as_str(), "Committee must have at least 4 members");
```

Comment:
```suggestion
            assert_eq!(err.to_string().as_str(), "Committee must have at least 3 members");
```
see the earlier comment